omod/src/main/webapp/resources/scripts/visit/visit.js

```diff
@@ -730,6 +732,7 @@ angular.module("visit", [ "filters", "constants", "encounterTypeConfig", "visitS
                         returnUrl: encodeURIComponent(
                           window.location.pathname
                           + "?visit=" + $scope.visit.uuid
+                          + "&encounter=" + encounter.uuid
```

Comment:
Fixed a bug where when you clicked "Cancel" while edited a form you lost the "Saved Form" context